### PR TITLE
Fix lint error

### PR DESCRIPTION
**Pull request by Kourin1996**
Fix EDGE-435
Fix lint error by solhint without the change of functionality

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "test": "npx hardhat test",
     "clean": "npx hardhat clean",
     "compile": "npx hardhat compile",
-    "lint": "solhint 'contracts/*.sol'"
+    "lint": "solhint 'contracts/**/*.sol'"
```
Comment (Aleksao998) [inline package.json:17]: Great catch

**Review by Aleksao998 — APPROVED** [review]
LGTM!

**Review by zivkovicmilos — APPROVED** [review]
Ran the linter, everything passes 👍

**Review by lazartravica — APPROVED** [review]

**Comment by GitesParti343** [comment]
7b42f57

**Comment by GitesParti343** [comment]
ii

**Pull request merged**